app/main.ipynb

```diff
@@ -192,10 +239,19 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
-   "outputs": [],
+   "execution_count": 33,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[2.  8.4]\n"
+     ]
+    }
+   ],
    "source": [
-    "# write your code here"
+    "column_means = np.mean(A, axis=0)\n",
```

Comment:
np.mean(A, axis=0) finds mean by Y axis. You must use np.mean(A, axis=1) to find the mean for each column by X axis

Reply:
Are u right? 
<img width="742" alt="image" src="https://github.com/mate-academy/py-numpy/assets/108476061/17d92288-edcd-4e5c-9100-9a20442c42e0">

Reply:
I'm sure. 
![image](https://github.com/mate-academy/py-numpy/assets/129166823/45d3489a-d2f4-4b63-971d-6e1432207f8a)

Reply:
I agree with u